nvidia_triton/datadog_checks/nvidia_triton/metrics.py

```diff
@@ -0,0 +1,37 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+METRICS_MAP = {
+    'nv_cache_insertion_duration': 'nv.cache.insertion.duration',
+    'nv_cache_lookup_duration': 'nv.cache.lookup.duration',
+    'nv_cache_num_entries': 'nv.cache.num.entries',
+    'nv_cache_num_evictions': 'nv.cache.num.evictions',
+    'nv_cache_num_hits': 'nv.cache.num.hits',
+    'nv_cache_num_lookups': 'nv.cache.num.lookups',
+    'nv_cache_num_misses': 'nv.cache.num.misses',
+    'nv_cache_util': 'nv.cache.util',
+    'nv_cpu_memory_total_bytes': 'nv.cpu.memory.total_bytes',
+    'nv_cpu_memory_used_bytes': 'nv.cpu.memory.used_bytes',
+    'nv_cpu_utilization': 'nv.cpu.utilization',
+    'nv_energy_consumption': 'nv.energy.consumption',
+    'nv_gpu_memory_total_bytes': 'nv.gpu.memory.total_bytes',
+    'nv_gpu_memory_used_bytes': 'nv.gpu.memory.used_bytes',
+    'nv_gpu_power_limit': 'nv.gpu.power.limit',
+    'nv_gpu_power_usage': 'nv.gpu.power.usage',
+    'nv_gpu_utilization': 'nv.gpu.utilization',
+    'nv_inference_compute_infer_duration_us': 'nv.inference.compute.infer.duration_us',
+    'nv_inference_compute_infer_summary_us': 'nv.inference.compute.infer.summary_us',
+    'nv_inference_compute_input_duration_us': 'nv.inference.compute.input.duration_us',
+    'nv_inference_compute_input_summary_us': 'nv.inference.compute.input.summary_us',
+    'nv_inference_compute_output_duration_us': 'nv.inference.compute.output.duration_us',
+    'nv_inference_compute_output_summary_us': 'nv.inference.compute.output.summary_us',
+    'nv_inference_count': 'nv.inference.count',
+    'nv_inference_exec_count': 'nv.inference.exec.count',
+    'nv_inference_pending_request_count': 'nv.inference.pending.request.count',
+    'nv_inference_queue_duration_us': 'nv.inference.queue.duration_us',
+    'nv_inference_queue_summary_us': 'nv.inference.queue.summary_us',
+    'nv_inference_request_duration_us': 'nv.inference.request.duration_us',
+    'nv_inference_request_summary_us': 'nv.inference.request.summary_us',
+    'nv_inference_request_failure': 'nv.inference.request_failure',
+    'nv_inference_request_success': 'nv.inference.request_success',
```

Comment:
Is the `nv` prefix really needed since all our metrics have it (in addition to our default prefix `nvidia_triton`)?

Reply:
good point, making changes.